tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

```diff
@@ -572,6 +573,19 @@ def test_task_two_granules(self, mock_debug: MagicMock):
         # Validate the output is correct by matching with the output schema
         _OUTPUT_VALIDATE(exp_result)
 
+    @patch("extract_filepaths_for_granule.LOGGER.debug")
+    def test_task_use_recovery_override_bucket(self, mock_debug: MagicMock):
+        """
+        Test no 'granules' key in input event.
+        """
+        self.task_input_event[
+            "input"]["granules"][0]["recoverybucketoverride"] = uuid.uuid4().__str__()
+
+        result = extract_filepaths_for_granule.task(self.task_input_event)
+        self.assertEqual(
+            result["granules"][0]["keys"][0]["destBucket"],
+            self.task_input_event["input"]["granules"][0]["recoverybucketoverride"])
```

Comment:
Wonder if we shouldn't use our defined global key name variables throughout instead. For example:
`result["granules"][0]["keys"][0]["destBucket"]` becomes `result["granules"][0][OUTPUT_KEY_KEY][0][OUTPUT_DESTINATION_BUCKET_KEY]` @AndrewDorn  thoughts?

Reply:
Agreed

Reply:
yes should probably use the later to stay consistent with other unit tests.
 OUTPUT_KEY_KEY refers to "key" but in this case we want "keys" array
 
Changed to 

```
        self.assertEqual(
            result["granules"][0]["keys"][0][
                extract_filepaths_for_granule.OUTPUT_DESTINATION_BUCKET_KEY
                ], self.task_input_event["input"]["granules"][0][
                    extract_filepaths_for_granule.INPUT_RECOVERY_BUCKET_OVERRIDE_KEY])
```

Reply:
Ah, I only noticed the change to OUTPUT_DESTINATION_BUCKET_KEY. Yeah, a lot of our input stuff is still decentralized. Looks good.